index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <h2 class="logo">my logo</h2>
```

Comment:
- [ ] Kindly use `a` tag for your logo as requested in the project requirement. change `<h2 class="logo">my logo</h2>` to `<a href="#" class="logo">my logo</h2>`